release.yaml

```diff
@@ -42,6 +42,7 @@ rancher-pushprox:
 rancher-vsphere-csi:
 - 2.3.1
 - 100.1.1+up2.3.0-rancher1
```

Comment:
Please remove the 100.1.1+up2.3.0-rancher1 entry and its assets, 100.2.0+up2.4.1-rancher1 replaces it.

Reply:
I made the changes can you please review it again?